### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by Bludidi**
## Video Presentation
Checkout the [video here](https://drive.google.com/file/d/1m-Ib7AopeDt2JfONw1M7u2KCm658cAwV/view?usp=share_link)
## In this pull request we have implemented the following:

 - Created `Item class` in a separate .rb file.
 - Added all methods visible in the diagram.
 - Created a `main.rb` file that will serve as your console app entry-point.
 - Implement startup actions
 - Created a `Book class` in a separate .rb file.
 - Created a `Label class` with an association to the Item class (in a separate .rb file).
 - All `Book class` properties visible in the diagram were defined and set up in the constructor method.
 - All `Label class` properties visible in the diagram were defined and set up in the constructor method.
 - Added unit tests for all implemented methods
 - Preserved all data in `json` files
 - Created `MusicAlbum class` in a separate .rb file.
 - Created `Genre class` with an association to the `Item class` (in a separate .rb file).
 - All `MusicAlbum class` properties visible in the diagram were defined and set up in the constructor method.
 - All `Genre class` properties visible in the diagram were defined and set up in the constructor method.
 - Created a `Game class` in a separate .rb file.
 - Created an `Author class` with an association to the Item class (in a separate .rb file).
 - All `Game class` properties visible in the diagram were defined and set up in the constructor method.
 - All `Author class` properties visible in the diagram were defined and set up in the constructor method.
 - Created a `schema.sql` file with tables that will be analogical to the structure of the classes that you created:
    - games table (add all properties and associations from the parent Item class as table columns)
authors table

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,176 @@
...
+```sh
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command in your terminal:
+
+```sh
+  ruby src/main.rb
```
Comment (danielufeli) [inline README.md:94]: - [ ] Nice work done on your Readme, kindly change this `ruby main.rb` it throws an error when I used `ruby src/main.rb`.

File: src/game.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+require_relative './item'
+require 'Date'
```
Comment (danielufeli) [inline src/game.rb:2]: Nice working your game class, kindly change the `require 'Date'` to `require 'date'`, its throws the error in the screenshot below when you use `require 'Date'`.
--Date-Error

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @Team, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights
- Nice commits messages✔️
- Clean Code ✔️

## Suggested changes
- [ ] Great work done on your project and video presentation team, has a requirement the  maximum length of your video should be 5 minutes kindly reduce yours from 9.41mins to 5mins, and all members should talk about the project in the video.

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (Bludidi) [reply README.md]: Hi @danielufeli we have updated the command as you suggested.

File: src/game.rb (hunk 2 above)
Comment (Bludidi) [reply src/game.rb]: We have changed `Date` to `date` also

**Review by EricMbouwe — APPROVED** [review]
Hi @Bludidi, 

In my opinion, your project is ready for presentation! ⚡🔥 There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**